tuxemon/core/db.py

```diff
@@ -86,11 +75,22 @@ def load(self, directory="all"):
 
         """
 
+        self.database = {
```

Comment:
why is this moved?

Reply:
I think I misunderstood one of our conversations on Discord. Placed it back in the last commit.